app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  class BaseController < ApplicationController
+    protect_from_forgery with: :null_session
+
+    skip_before_action :set_current_team
+    skip_before_action :run_first_time_setup
+    skip_before_action :ensure_team_exists_for_subdomain!
+    skip_before_action :require_login
+    skip_before_action :user_belongs_to_team!
+
+    before_action :require_api_token
+    before_action :set_current_team
+
+    private
+
+    def render_errors(object)
+      render json: {message: "Validation Failed", errors: object.errors.full_messages}, status: 422
```

Comment:
i wonder if we should just return the array of errors `object.errors`, i've found it handy a couple of times to know what field certain errors are on, and then being able to add the errors directly on the fields